react/Footer/FooterLink/FooterLink.js

```diff
@@ -36,9 +38,16 @@ FooterLink.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node
   ]),
-  linkRenderer: PropTypes.func.isRequired
+  linkRenderer: PropTypes.func.isRequired,
+  authedHref: PropTypes.string,
```

Comment:
I'm open to suggestions however in this case it's not just setting the returnURL from /sign-in it's also adding the param `nudge=saved` so `https://www.seek.com.au/sign-in?nudge=saved`.